api/migrations/Version20230530142035.php

```diff
@@ -27,7 +27,7 @@ public function up(Schema $schema): void
         $this->addSql('CREATE INDEX IDX_A5E2A5D7115F0EE5 ON configuration (domain_id)');
         $this->addSql('CREATE TABLE domain (id INT NOT NULL, owner_id INT NOT NULL, dns VARCHAR(255) NOT NULL, valid BOOLEAN NOT NULL, PRIMARY KEY(id))');
         $this->addSql('CREATE INDEX IDX_A7A91E0B7E3C61F9 ON domain (owner_id)');
-        $this->addSql('CREATE TABLE "user" (id INT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, lastname VARCHAR(100) NOT NULL, firstname VARCHAR(100) NOT NULL, company VARCHAR(150) DEFAULT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE TABLE "user" (id INT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, lastname VARCHAR(100) NOT NULL, firstname VARCHAR(100) NOT NULL, company VARCHAR(150) DEFAULT NULL, token VARCHAR(530) DEFAULT NULL, activated BOOLEAN NOT NULL, PRIMARY KEY(id))');
         $this->addSql('CREATE UNIQUE INDEX UNIQ_8D93D649E7927C74 ON "user" (email)');
         $this->addSql('ALTER TABLE configuration ADD CONSTRAINT FK_A5E2A5D7115F0EE5 FOREIGN KEY (domain_id) REFERENCES domain (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
         $this->addSql('ALTER TABLE domain ADD CONSTRAINT FK_A7A91E0B7E3C61F9 FOREIGN KEY (owner_id) REFERENCES "user" (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
```

Comment:
Why are you renaming the existing migration? You're breaking the Symfony migration system with that.
A committed migration must not be renamed but you have to create a new one that contains the new columns.

Reply:
I did not renamed it, i cleared all the migration in the migrations directory and in the database then I re generated a new one will all the property

Reply:
That's what I said, you should not do that like that.

Reply:
How can i restore it ?

Reply:
Revert the changes